packages/ui-components/src/components/UISearchBox/UISearchBox.tsx

```diff
@@ -30,7 +30,6 @@ export class UISearchBox extends React.Component<ISearchBoxProps, {}> {
             ...{
                 root: [
                     {
-                        fontFamily: 'var(--vscode-font-family)',
```

Comment:
and probably here as well (if search input doesn't inherit from some common input component).

Reply:
as checked - I do not see reason to leave it, same explanation as - https://github.com/SAP/open-ux-tools/pull/3060#discussion_r2020847135